.github/workflows/fuzz-test.yml

```diff
@@ -27,11 +27,6 @@ jobs:
 
       - name: Checkout Code
         uses: actions/checkout@v3
-        with:
-          submodules: recursive
-
-      - name: Install Dependencies
-        run: ./setup-ci.sh
```

Comment:
Why is this removed? What if want to run fuzz test independently?

Reply:
Glad that you have asked, somehow I felt you were going to ask me these kinds of questions. :smile: 

> Why is this removed?

It is removed because it is not needed and it has a negative aspect on CI duration.

> What if want to run fuzz test independently?

You are able to do so because `fuzz-test.yml` is in fact independent GH workflow.

Joke on the side, as you can see I have manually triggered `fuzz-tests` here and it was successfully executed
(https://github.com/0xPolygon/polygon-edge/actions/runs/4765173994).

Now to explain what I did here:
- we don't need git submodules in place for fuzz testing and that is why I've removed `submodules: recursive`
- we don't need a Solidity compiler to be installed on the CI for the fuzz tests and that is why I have removed these two lines:

```yml
 - name: Install Dependencies
   run: ./setup-ci.sh
```